app/notifications/notifications.module.ts

```diff
@@ -9,6 +9,8 @@ import { notificationRouting } from './notifications.routing';
 import { NotificationsComponent } from './notifications.component';
 import { NotificationsListComponent } from './notifications-list/notifications-list.component';
 import { NotificationService } from './notification.service';
+import { NotificationsFilter } from './pipes/notification-filter.pipe';
+
```

Comment:
CG: Please include `NotificationsFilter` in alphabetical order. Don't add a new empty line here.